index.js

```diff
@@ -9,15 +10,16 @@ module.exports = function (element, options) {
   function inject (element) {
     var parentNode = element.parentNode
     var string = element.nodeValue
-    var length = string.length
+    var split = splitRegex ? string.split(splitRegex) : string
+    var length = split.length
     var i = -1
-    while (++i < length) {
+    while (++i < length)  {
```

Comment:
I'm just curious, what's the benefit of using while instead of for or map in this case?

Reply:
No benefit other than perhaps that it's concise. But really it’s just a convention in this code base for doing loops

Reply:
Ah! I find for loop with inline var because garbage collector will automatically collect that variable at the end. Therefore, no residue. Nowadays I use map instead of for if I don't have to break it in middle somewhere which is more efficient. and gives me more control over data. :)